src/commands/synthetics/__tests__/cli.test.ts

```diff
@@ -1428,6 +1430,34 @@ describe('upload-application', () => {
       })
     })
   })
+
+  describe('any kind of error is reported', () => {
+    test.each([
+      [
+        'CI error',
+        new CriticalError('INVALID_MOBILE_APP', 'some message'),
+        'A CI error occurred: [INVALID_MOBILE_APP] some message',
+      ],
+      ['Endpoint error', new api.EndpointError('some message', 404), 'A backend error occurred: some message (404)'],
+      [
+        'Axios error',
+        getAxiosError(400, {message: 'Bad Request'}),
```

Comment:
This is an unexpected error because we should have a wrapper around all axios calls, which translates the Axios error into an `EndpointError`.